src/psyclone/tests/psyir/transformations/hoist_trans_test.py

```diff
@@ -153,12 +153,12 @@ def test_validate_error_read_and_write(fortran_reader, assignment_str):
 
     '''
     psyir = fortran_reader.psyir_from_source(
-        '''subroutine test()
+        f'''subroutine test()
               integer :: i, j, a, b(10)
               do i=1, 10
-                  {0}
+                  {assignment_str}
               enddo
-              end subroutine test'''.format(assignment_str))
```

Comment:
I kept the indentation as it was, but I can't help but feel it's a bit weird, e.g. `end subroutine` has the same indentation has the subroutine's body....

Reply:
Fixed it myself for now (unless I find bigger issues to request changes)